### ISSUE TITLE
Changing Xuggler to MonteMedia Library

### ISSUE THREAD

**Issue by lordofthejars** [arquillian/arquillian-recorder#23]
Arquillian Desktop Video Recorder cannot be released because of license of Xuggler. Change `VideoRecorder` to use MonteMedia library which is released with 3 licenses CC, GPL and APL2.

**Comment by MatousJobanek** [comment]
Hi,
great that it has been changed :+1: 
I guess that we can remove the skip in the maven deployment plugin, right?
https://github.com/arquillian/arquillian-recorder/blob/master/arquillian-desktop-video-recorder/pom.xml#L82-L89

**Comment by lordofthejars** [comment]
Yes, now there is no license problems. What I would like to find (and I have tried and I couldn't) is the way to set JCodec to make frames at the required fps. But probably this is another story.

**Comment by bartoszmajsak** [comment]
Can we close this issue @lordofthejars ?
